src/views/Users/ViewUser/index.jsx

```diff
@@ -376,7 +492,7 @@ function ViewUser({ isNewUser, ...props }) {
                 disabled={saveAction.loading}
                 icon={<DeleteIcon />}
                 tooltipOpen
-                tooltipTitle="Delete Signoff"
+                tooltipTitle="Delete User"
                 onClick={handleUserDelete}
```

Comment:
It would be nice if we can have a confirmation dialog for destructive actions. OK to do in a follow-up if that's easier. Note: `saveAction.loading` doesn't seem to be used in `isLoading` which is okay if we're using a DialogAction since the latter has support for showing a spinner inside a dialog for asynchronous actions.

Reference: https://github.com/mozilla-frontend-infra/balrog-ui/issues/20

Reply:
I agree, we should do this in a follow-up.